app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -365,6 +367,19 @@ class MessagingManager @Inject constructor(
                             }
                         }
                     }
+                    COMMAND_BEACON_MONITOR -> {
+                        if (!jsonData[TITLE].isNullOrEmpty() && jsonData[TITLE] in ENABLE_COMMANDS)
```

Comment:
Now that #1759 has been merged we will need to update this PR to switch from `TITLE` to `COMMAND`, we shoudl also update the documentation to reflect this change as well.